docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>avec xclim 0.47, cette cellule brise parce que change_significance a une nouvelle signature. </p><p><a href="https://xclim.readthedocs.io/en/stable/changes.html" target="_blank">https://xclim.readthedocs.io/en/stable/changes.html</a></p><p>change_significance va m[eme disparaitre à 0.49. but this can be fixed in another PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>